bin/tls-include.sh

```diff
@@ -256,8 +265,14 @@ function create_tls_certs_openssl {
       secretName="${app}-tls-secret"
 
       if [ -n "$(kubectl get secret -n $namespace $secretName -o name 2>/dev/null)" ]; then
-         log_debug "TLS Secret for [$app] already exists; skipping TLS certificate generation."
-         continue
+        if kubectl get secret -n $namespace $secretName -o "jsonpath={.data['tls\.crt']}" | base64 -d | openssl x509 -checkend $DAY_IN_SECONDS -noout 2>/dev/null
```

Comment:
See comments above since they apply here as well.  I wonder if there's value in creating a function to determine whether cert is expired.  That would allow us to call the same function in both places and allow us to update the expiration criteria in a single place.  

In any case, let's use a variable name like CERT_EXPIRATION rather than DAY_IN_SECONDS, since we may, at some point, decide we want the expiration period to be something other than 24 hours.

Hmmmm, if we have different functions for cert-manager vs. openssl, maybe we can implement more sophisticated logic that takes into account cert-manager's ability to regenerate certs.  Let's discuss off-line.